kickstarter/android-oss
# Leigh/nt 1442 error state shipping selector

# 📲 What

We want to hide the shipping selector when there is a network call error to fetch addons, and only show when addons are fetched correctly. 

# 🛠 How

Update `shippingSelectorGone` stream when there is a network error. 

# 👀 See

![shippingSelectorGone](https://user-images.githubusercontent.com/19390326/94033645-2c15d380-fd8f-11ea-8079-84233535ecd8.gif)

# 📋 QA

For error alert dialog:

- Search for the project "fadoodle"
- Select "back project"
- Turn off wifi or put phone in airplane mode
- Select a pledge with add-ons
- Upon error dialog popping up, location selector should disappear
- Before tapping "retry", turn wifi on or take phone off airplane mode
- Press "retry"
- Network call should be successful and location selector should reappear

# Story 📖

[Android Display an error state if add-ons fail to load
](https://kickstarter.atlassian.net/browse/NT-1442)
